### PR TITLE
UnixPB: Include installation of docker-buildx plugin for Ubuntu 20+

### LINKED ISSUE #3531: Ansible request for docker-buildx plugin.
For everywhere we use docker, the docker-buildx plugin is required for docker-build to work correctly, on more recent versions of docker.

Delete as appropriate from this list:

- Missing install

Details:

When running docker builds for the installers,  on all the new rise machines ( and this was previously seen on other dockerhosts and worked around by manually installing the missing component.

```
ERROR: BuildKit is enabled but the buildx component is missing or broken.
       Install the buildx component to build images with BuildKit:
       https://docs.docker.com/go/buildx/
```

### PR DESCRIPTION
**Pull request by steelhead31**
Fixes #3531 

Include the installation of docker-buildx plugin , required for installer builds and docker builds in general.

##### Checklist

- [X] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [X] playbook changes run through [VPC](https://ci.adoptium.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptium.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)

### CODE REVIEW

**Comment by steelhead31** [comment]
VPC in progress : https://ci.adoptium.net/job/VagrantPlaybookCheck/1869/

**Review by karianna — APPROVED** [review]

File: .github/workflows/build_vagrant.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
     - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
     - name: Update Repos
-      run: sudo apt-get update
+      run: sudo apt-get update || true
```
Comment (sxa) [inline .github/workflows/build_vagrant.yml:28]: Why this change?

**Review by sxa — CHANGES_REQUESTED** [review]

File: .github/workflows/build_vagrant.yml (hunk 1 above)
Comment (steelhead31) [reply .github/workflows/build_vagrant.yml:28]: Temporary fix for the broken ubuntu runner action... I forgot to remove it... I'll do it now

**Review by sxa — APPROVED** [review]
